pythonforandroid/graph.py

```diff
@@ -133,13 +271,14 @@ def get_recipe_order_and_bootstrap(ctx, names, bs=None):
             raise
         orders.append(list(order))
 
-    # prefer python2 and SDL2 if available
+    # prefer python3 and SDL2 if available
     orders = sorted(orders,
-                    key=lambda order: -('python2' in order) - ('sdl2' in order))
+                    key=lambda order: -('python3' in order) - ('sdl2' in order))
 
     if not orders:
         raise BuildInterruptingException(
-            'Didn\'t find any valid dependency graphs. This means that some of your '
+            'Didn\'t find any valid dependency graphs. '
+            'This means that some of your '
             'requirements pull in conflicting dependencies.')
```

Comment:
Thanks for sneaking `E501` on some of your pull requests :pray:
I actually also respect it on my projects, but p4a had different history so far :smile:

Reply:
thanks :heart: I'm resolving this conversation so we get a better view of what's still left to do